Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -10931,7 +10935,7 @@ private command __tryToDoScriptInObject pObject, pToDo, @rValidScript
    put the result into tError
    if tError is empty then
       try
-         send "-- execute in object context" & return & tDoScript to pObject
+         send tDoScript to pObject
```

Comment:
The extra bit of comment here is to make sure that the engine uses `domess` instead of calling a handler in the target object directly - without it, I believe if you do something like:

```
myHandler "Param1", "Param2"
myOtherHandler "Param1", "Param2"
```
this will cause `myHandler` to be called with three parameters:
1. "Param1"
2.  "Param2"
myOtherHandler "Param1"
3. "Param2"